internals/overlord/logstate/loki/loki.go

```diff
@@ -38,16 +38,17 @@ const (
 )
 
 type Client struct {
-	options *ClientOptions
-
-	targetName string
-	remoteURL  string
+	options    *ClientOptions
+	target     *plan.LogTarget
 	httpClient *http.Client
 
 	// To store log entries, keep a buffer of size 2*MaxRequestEntries with a
 	// sliding window 'entries' of size MaxRequestEntries
 	buffer  []lokiEntryWithService
 	entries []lokiEntryWithService
+
+	// store the custom labels for each service
+	labels map[string]map[string]string
```

Comment:
Can you please extend the comment to indicate what each key does.
`// example usage: labelValue := labels[serviceName][labelKey]`